lua/nvim-tree/colors.lua

```diff
@@ -33,6 +33,7 @@ local function get_hl_groups()
   return {
     IndentMarker = { fg = "#8094b4" },
     Symlink = { gui = "bold", fg = colors.cyan },
+    SymlinkIcon = { fg = colors.cyan },
```

Comment:
Unfortunately we can't apply this default as it will break existing users.

SymlinkIcon should only be applied when the user defines it.